website/docs/developer/deployment-guide/deploying-from-windows.mdx

```diff
@@ -0,0 +1,185 @@
+The goal of this page is to provide a set of modified instructions to get around common errors.
+
+### Notes
+- Choose a PREFIX that will identify your installation when in AWS. This string will be used throughout deployment.
+- Connect to the NASA VPN to be able to connect to AWS.
+  :::warning
+  The VPN drastically slows down Terraform operations, and limits what documentation can be viewed. Switch off when applicable.
+  :::
+- Commands here will use ```us-west-2``` for region because of the current state of our sandbox. Replace consistently as needed.
+    - Make sure any operations in AWS are done under the correct region.
+
+## Application
+This application will be used in future steps to authenticate users.
+- Go to https://uat.urs.earthdata.nasa.gov/profile
+- Applications -> My Applications
+- Create a new Application
+    - Application ID: ```prefix_cumulus```
+    - Application Name ```Prefix Cumulus```
+    - Application Type: ```OAuth 2```
+    - Redirect URL: todo
+
+## Initial Setup
+- Follow the [deployment environment setup instructions](setting-up-deployment-environment.mdx).
+    - You may need to install Terraform manually.
+    - Only configure the default profile.
+    - Keep the access keys in plain-text. You will need to run ```aws configure``` in multiple environments.
+- Create an AWS Key Value Pair by following  [the AWS instructions](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/ec2-key-pairs.html#having-ec2-create-your-key-pair).
+    - Choose the '.pem' format.
+    - Naming convention is PREFIX-key-pair.pem
+- Create buckets in the same OU and region as the other installations.
+    - Same OU and region would not be ideal for a real backup system, but is generally sufficient for testing.
+    - Required buckets are PREFIX-tf-state, PREFIX-glacier, PREFIX-internal, PREFIX-private, PREFIX-protected, PREFIX-public, and PREFIX-orca
+    :::tip
+    An example command for creating a bucket in us-west-2. Remember to run ```aws configure``` first.
+    ```commandline
+    aws s3api create-bucket --bucket PREFIX-tf-state --profile default --region us-west-2 --create-bucket-configuration "LocationConstraint=us-west-2"
+    ```
+
+## [Cumulus](https://nasa.github.io/cumulus/docs/deployment/deployment-readme)
+- If creating a realistic setup with multiple OUs, apply [Create the ORCA Archive Bucket](creating-orca-glacier-bucket.md) to your PREFIX-glacier.
+- Run
+  ```commandline
+  aws dynamodb create-table --table-name PREFIX-tf-locks --attribute-definitions AttributeName=LockID,AttributeType=S --key-schema AttributeName=LockID,KeyType=HASH --billing-mode PAY_PER_REQUEST --region us-west-2
+  ```
+- Run
+  ```commandline
+  aws s3api put-bucket-versioning --bucket PREFIX-tf-state --versioning-configuration Status=Enabled
+  ```
+  :::warning
+  We presently have no instructions for creating a VPC, Subnet, Security Group, or PostgreSQL database.
```

Comment:
I don't know what it means that "The VPC is created by NGAP". I would want a link to how/why that is automatically created. Is that for any AWS account? What will happen when we deploy to Localstack?

If the subnets are created by Terraform, then it can't be this Terraform; rds-cluster-tf requires those values, and that's the first module deployed.

Assuming that the terraform that creates the database/security group is the TF that's run in these instructions, I can remove those comments.

Reply:
So when NGAP creates and OU environment like sandbox for Cumulus it creates a single VPC that we all use for that environment. In other words we get handed the VPC. We do not create it same with the private subnets. Information on both are available here https://wiki.earthdata.nasa.gov/display/ESKB/AWS+Services+Approval+Status+Page#AWSServicesApprovalStatusPage-VPC

I believe all the security groups are created in terraform for cumulus and orca. See the following examples:
- https://github.com/nasa/cumulus/blob/2ce5a65fbea48ccd02a97a0ae837380cabec5bd4/tf-modules/cumulus-rds-tf/main.tf#L22
- https://github.com/nasa/cumulus/blob/2ce5a65fbea48ccd02a97a0ae837380cabec5bd4/tf-modules/cumulus/ecs_cluster.tf#L181
- https://github.com/nasa/cumulus-orca/blob/master/modules/security_groups/main.tf

Reply:
Alright. Downgraded to tip and added link. That could make automated integration testing interesting.